lottie-swift/src/Private/Model/Extensions/Bundle.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+#if os(iOS) || os(tvOS) || os(watchOS) || targetEnvironment(macCatalyst)
+import UIKit
+#endif
```

Comment:
usually I just use `#if canImport(UIKit)` but this is the pattern used elsewhere in the project, so I felt it was best to be consistent